File: src/main/java/DaysJava/week1/Day6SentenceCap.java
package DaysJava.week1;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Day6SentenceCap {

public static void main(String[] args){
    Day6SentenceCap sentenceCap = new Day6SentenceCap();
    System.out.println("Enter the sentence to capitalize: ");
    String sentence = new Scanner(System.in).nextLine();
    sentenceCap.sentenceCapitalization(sentence);
}
    private void sentenceCapitalization(String sentence) {
    String[] splitSentence = sentence.split(" ");
        List<String> capitalizeWords = new ArrayList<>();
        for(String words : splitSentence) {
            capitalizeWords.add(Character.toUpperCase(words.charAt(0)) + words.substring(1));
        }
        System.out.println(String.join(" ", capitalizeWords));
    }
}

File: src/main/kotlin/DaysKotlin/week1/Day6SentenceCap.kt
package DaysKotlin.week1

import java.util.*

fun main(){
println("Enter the sentence to capitalize: ")
    val sentence = Scanner(System.`in`).nextLine()
    sentenceCapitalization(sentence)
}

fun sentenceCapitalization(sentence: String){
        var splitSentence = sentence.split(" ").toMutableList()
    var capitalizeSentence = ""
        for(i in splitSentence){
            capitalizeSentence +=i.capitalize() + " "
        }
    capitalizeSentence.trim()

    print(capitalizeSentence)
}